x-pack/plugins/maps/public/components/toolbar_overlay/_index.scss

```diff
@@ -8,6 +8,7 @@
 .mapToolbarOverlay__button {
```

Comment:
Can you do me a favor and change the top calculation of `.mapToolbarOverlay` in this file to:
```scss
top: ($euiSizeM + $euiSizeS) + ($euiSizeXL * 2); // Position and height of mapbox controls plus margin
```